kubernetes/bkvm/templates/deployment.yaml

```diff
@@ -0,0 +1,57 @@
+#
+# Licensed to Diennea S.r.l. under one
+# or more contributor license agreements. See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership. Diennea S.r.l. licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bkvm-server
+  labels:
+    {{ toYaml .Values.server.labels | nindent 4 }}
+spec:
+  replicas: 1
```

Comment:
this should be configurable

Reply:
I don't think that BKVM supports multiple instances at the moment (because there is no handling of distributed sessions).
I think that we should keep 1 here

Reply:
I didn't test it with several replicas but one replica works with Pulsar